spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/EventProcessorProperties.java

```diff
@@ -89,6 +91,16 @@
          */
         private int initialSegmentCount = 1;
 
+        /**
+         * See {@link TrackingEventProcessorConfiguration#andTokenClaimInterval(long, java.util.concurrent.TimeUnit)}
```

Comment:
This piece of javadoc will be shown to users on a configuration file as well. I do not feel that a link to another piece of javadoc is fair there. Would you mind adjusting this to actually explain what setting this field means?

Additionally, pointing out what the default is here is also very valuable to users.

Reply:
@smcvb I do agree.